client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-definitions/index.ts

```diff
@@ -16,14 +16,13 @@ const DEFINITIONS = {
 	...planActions,
 } satisfies TaskActionTable;
 
-const MIGRATED_FLOWS = [ 'free' ];
+export const FLAG_NAME = 'launchpad/new-task-definition-parser';
```

Comment:
Nit: this constant name is not very descriptive. What is the flag used for? What does it do? As an example, calling it `NEW_TASK_DEFINITION_PARSER_FEATURE_FLAG` makes it _much_ clearer what the flag controls, just by reading the name. (We could use something else, too, this is mostly meant to be an illustration/suggestion.) 
```suggestion
export const NEW_TASK_DEFINITION_PARSER_FEATURE_FLAG = 'launchpad/new-task-definition-parser';
```

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/86594/commits/04d7c859eb458693c33d8ffb6018594d0750a712